extras/rya.export/export.accumulo/src/main/java/org/apache/rya/export/accumulo/AccumuloParentMetadataRepository.java

```diff
@@ -29,15 +61,210 @@
+     */
+    private static final Text MERGE_PARENT_METADATA_RYA_INSTANCE_NAME = new Text("ryaInstanceName");
+
+    /**
+     * The Column Qualifier for the timestamp.
+     */
+    private static final Text MERGE_PARENT_METADATA_TIMESTAMP = new Text("timestamp");
+
+    /**
+     * The Column Qualifier for the copy tool input time.
+     */
+    private static final Text MERGE_PARENT_METADATA_COPY_TOOL_INPUT_TIME = new Text("copyToolInputTime");
+
+    /**
+     * The Column Qualifier for the copy tool run time.
+     */
+    private static final Text MERGE_PARENT_METADATA_COPY_TOOL_RUN_TIME = new Text("copyToolRunTime");
+
+    /**
+     * The Column Qualifier for the parent time offset.
+     */
+    private static final Text MERGE_PARENT_METADATA_PARENT_TIME_OFFSET = new Text("parentTimeOffset");
+
+    // Lexicoders used to read/write MergeParentMetadata to/from Accumulo.
+    private static final LongLexicoder LONG_LEXICODER = new LongLexicoder();
+    private static final StringLexicoder STRING_LEXICODER = new StringLexicoder();
+    private static final DateLexicoder DATE_LEXICODER = new DateLexicoder();
+
+    private static final String BASE_MERGE_METADATA_TABLE_NAME = "merge_metadata";
+
+    private final AccumuloRyaDAO accumuloRyaDao;
+    private final Connector connector;
+    private final String tablePrefix;
+    private final String mergeParentMetadataTableName;
+
+    /**
+     * Creates a new instance of {@link AccumuloParentMetadataRepository}.
+     * @param accumuloRyaDao the {@link AccumuloRyaDAO}. (not {@code null})
+     */
+    public AccumuloParentMetadataRepository(final AccumuloRyaDAO accumuloRyaDao) {
+        this.accumuloRyaDao = checkNotNull(accumuloRyaDao);
+        connector = accumuloRyaDao.getConnector();
+        tablePrefix = accumuloRyaDao.getConf().getTablePrefix();
+        mergeParentMetadataTableName = tablePrefix + BASE_MERGE_METADATA_TABLE_NAME;
+    }
 
     @Override
     public MergeParentMetadata get() {
-        return metadata;
+        checkNotNull(connector);
+        checkNotNull(mergeParentMetadataTableName);
+
+        try {
+            // Create an Accumulo scanner that iterates through the metadata entries.
+            final Scanner scanner = connector.createScanner(mergeParentMetadataTableName, new Authorizations());
+            final Iterator<Entry<Key, Value>> entries = scanner.iterator();
+
+            // No metadata has been stored in the table yet.
+            if (!entries.hasNext()) {
+                log.error("Could not find any MergeParentMetadata metadata in the table named: " + mergeParentMetadataTableName);
+            }
+
+            // Fetch the metadata from the entries.
+            String ryaInstanceName = null;
+            Date timestamp = null;
+            Date copyToolInputTime = null;
+            Date copyToolRunTime = null;
+            Long parentTimeOffset = null;
+
+            while (entries.hasNext()) {
```

Comment:
Each entry is part of the same family but stored under different column qualifiers.